### ISSUE TITLE
Error when downloading one particular video

### ISSUE THREAD

**Issue by liotev** [rg3/youtube-dl#8109]
I get an error when trying to download one particular youtube video:

$youtube-dl https://www.youtube.com/watch?v=Zy6laZtRkcM
[youtube] Zy6laZtRkcM: Downloading webpage
....
  File "/opt/app//bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1884, in urlopen
  File "/usr/lib/python2.7/urllib2.py", line 396, in open
    protocol = req.get_type()
  File "/usr/lib/python2.7/urllib2.py", line 258, in get_type
    raise ValueError, "unknown url type: %s" % self.__original
ValueError: unknown url type: &asrs=1&type=list&tlangs=1

**Comment by georGEO1989** [comment]
You're running Python 2.7 have you tried update to Python 3?

**Comment by dstftw** [comment]
Post the **full** output of youtube-dl when run [with `--verbose` option](https://github.com/rg3/youtube-dl/blob/master/README.md#bugs).

**Comment by oittaa** [comment]
Probably very outdated Python. Works for me with Python 2 and Python 3 in Ubuntu 14.04.

**Comment by liotev** [comment]
I tried it with Python 3.5, the same thing happens.
But I found out that it happens only when my config file contains "--write-auto-sub". When I disable --write-auto-sub, the video downloads without a problem.
The verbose output is attached, in case anyone is interested. (I am on Mac OS X, with Python
 from Macports installation)
[youtube-dl-debug-output.txt](https://github.com/rg3/youtube-dl/files/81249/youtube-dl-debug-output.txt)
